shelter thingy/resources/JS/MegaMenu.js

```diff
@@ -0,0 +1,49 @@
+
+/*DeskNav*/
+
+$(document).ready(function () {
+    if ($(".NavHeight").backgroundColor === 'rgba(0,0,0, 0.5)') {
+        $(".NavHeight").mouseover(function () {
+            $(".NavHeight").animate({backgroundColor: 'rgba(0, 0, 0,.8)'});
+        });
+    }
+
+    $(".NavHeight").mouseout(function () {
+        $(".NavHeight").animate({backgroundColor: 'rgba(0, 0, 0,.5)'});
+    });
+});
+
+
+var count = 0;
+$(document).ready(function () {
+    $(".mega-dropdown").click(function () {
+        $(".mega-menu").slideToggle(slideTime);
+
+    });
+
+});
+
+function colorChange() {
+
+    count = (count + 1) % 2;
+    console.log(count);
+//ausklappen
+    if (count == 1 % 2) {
+        jQuery(".mega-menu").animate({backgroundColor: 'rgba(0, 0, 0, .8)'}, slideTime / 2);
+        /*     setTimeout(function () {
```

Comment:
Kommentare kannst du ruhig entfernen. 
Falls du es nochmal brauchst, wirst du die Infos in der git-History wiederfinden ;)

Reply:
erledigt.